src/lib.mjs

```diff
@@ -236,6 +237,16 @@ export const checkPage = async (args) => {
 
     await setTimeout(waitTimeMs)
 
+    if (mode === 'mhtml') {
```

Comment:
Instead of a new mode, can this work like the screenshot feature (i.e. the MHTML export is attached as an optional additional part of the request)? This would be much easier to use in a crawl context where we'd be interested in snapshotting the exact page where a cookie notice was detected.

Reply:
yeah, should be fine. Either you _need_ the MHTML export and so you opt-in to deal with the extra bandwidth required, or you don't need it and you don't request it in the first place.

On that note though, we should probably implement [`Content-Encoding`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) as well.

Reply:
opened https://github.com/brave/cookiemonster/issues/115